tests/unit/mpool/mblock_test.c

```diff
@@ -82,52 +73,52 @@ MTF_DEFINE_UTEST_PREPOST(mblock_test, mblock_abc, mpool_test_pre, mpool_test_pos
     ASSERT_EQ(ENOENT, merr_errno(err));
 
     err = mpool_info_get(mp, &info);
-    ASSERT_EQ(0, err);
+    ASSERT_EQ(0, merr_errno(err));
```

Comment:
Why is ASSERT_EQ(0, err) incorrect?

Reply:
Nabeel, previously I thought your style was to always do `ASSERT_EQ(0, merr_errno(err))`. So when I came across `ASSERT_EQ(0, err)`, I thought you had just forgotten to write `merr_errno(err)`. I realize now that you do it when you are only asserting an error code.

I adopted this style for the tests that I write because I find looking at the assertion as `0 == 22` instead of `0 == 1724301724876214768214` much easier to look at. But I can change it back. Like I said, only touched it in this file because I misunderstood your style.